frontend/src/lib/components/Uptodate.svelte

```diff
@@ -19,11 +24,5 @@
 </script>
 
 {#if uptodate}
-	<span class="text-blue-400"
-		>{uptodate} &nbsp;<Tooltip>
-			How to update?<br />
-			- docker: `docker compose up -d`<br />-
-			<a href="https://github.com/windmill-labs/windmill-helm-charts#install">helm</a>
-		</Tooltip>
-	</span>
+	<span class="text-blue-400">{uptodate} &nbsp;<Tooltip>{@html tooltipContent}</Tooltip></span>
```

Comment:
just inline the tooltipContent string here, no need for this

Reply:
@rubenfiszel fixed in latest commit